src/main/java/org/ballerinax/datamapper/DataMapperPlugin.java

```diff
@@ -363,7 +361,8 @@ private void processSampleDataFiles(String moduleName) {
                         Location position = new BLangDiagnosticLocation(path,
                                 startLocation.getLineNr() - 1, endLocation.getLineNr() - 1,
                                 startLocation.getColumnNr() - 1, endLocation.getColumnNr() - 1);
-                        dlog.logDiagnostic(DiagnosticSeverity.ERROR, position, "Error: Sample data provided for " +
+                        dlog.error(position, DiagnosticErrorCode.INCOMPATIBLE_TYPES,
+                                "Error: Sample data provided for " +
                                 typeName + " is different in terms of attributes count");
```

Comment:
This error gets logged as a compiler error right? Shall we check how this looks when logged in the terminal? Asking because we have a general format we follow for compile errors. For example,
1) Start the message with a lower case letter
2) Refrain from using prefixes like `Error:`
3) Use single quotes for type names, var names etc. e.g., `'Person'`

See https://github.com/ballerina-platform/ballerina-lang/blob/master/compiler/ballerina-lang/src/main/resources/compiler.properties as a reference

Reply:
Applies to the other places where errors are logged as well

Reply:
Fixed in [3e096a4](https://github.com/ballerina-platform/module-ballerinax-datamapper/pull/36/commits/3e096a4eb7a611475f39387c845e498230d53d7c)